### DISCUSSION TITLE
Is it possible to increase the files in $PROCESSED beyond 50?

### DISCUSSION THREAD

**Discussion by dfoster81** [mcguirepr89/BirdNET-Pi#707] [Q&A] [answered]
Apologies if this has been asked, I couldn't really see anything about it though. 

Just wondering if this is at all possible. I'm using an up to date image from #353 

I go for bushwalks (hiking but in Australia), for anywhere up to about 3 hours or so. It would just be neat to have more than the last 50 detection source files in the processed directory. The dawn chorus can get fairly hectic where I walk, so I wouldn't mind the original files to drop thru birdnet manually on different threshold settings to see if anything else pops up. 

Thanks!

**Comment by ehpersonal38** [comment, marked as answer]
Sure, in the file `~/BirdNET-Pi/scripts/cleanup.sh` you can edit lines 11 and 12 to change `100` (I think this is what you mean by 50, but Processed actually keeps closer to the last 100 files) to something larger, like `500`.

**Reply by dfoster81** [reply]
Thanks so much! Will try it tomorrow morning :)
